docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/pod-security-standards.md

```diff
@@ -1,20 +1,134 @@
+1. Configure the Pod Security Admission controller to suit your needs. Rancher allows you to configure this via [Pod Security Admission Configuration Templates](#psa-config-templates).
+    Review the templates and choose one of them, or create your own and apply to the clusters you are migrating.
+
+2. Map the PodSecurityPolicies that are still in use in your cluster into Pod Security Standards:
+    * You can find out which PodSecurityPolicies are still in use in your cluster by running
+      ```
+      kubectl get pods \
+        --all-namespaces \
+        --output jsonpath='{.items[*].metadata.annotations.kubernetes\.io\/psp}' \
+        | tr " " "\n" | sort -u
+      ```
+
+    * Follow the Kubernetes guide on [Mapping PodSecurityPolicy to Pod Security Standards](https://kubernetes.io/docs/reference/access-authn-authz/psp-to-pod-security-standards/) to apply PSS to your workloads that were relying on PSPs.
+
+      Learn more about migrating from PSPs to PSA and PSS on the [Migrate from PodSecurityPolicy to the Built-In PodSecurity Admission controller](https://kubernetes.io/docs/tasks/configure-pod-container/migrate-from-psp/) documentation.
+
+3. Remove PSPs from Rancher charts by upgrading them to the latest v102.x.y version _before_ your upgrade to Kubernetes v1.25.
+    Make sure the **Enable PodSecurityPolicies** option is **disabled**.
+    This will remove any PodSecurityPolicies that were installed with previous versions.
+
+:::info important
+If you intend to install the latest version, but do not plan on upgrading your clusters to Kubernetes v1.25 and moving away from PodSecurityPolicies, make sure that you select the option **Enable PodSecurityPolicies** for each chart you are upgrading to a v102.x.y version.
+:::
+
+### Cleaning up releases after a Kubernetes v1.25 upgrade
+
+If you have charts that did not provide a built-in mechanism to remove previously installed PSPs, or ran into other issues removing PSPs from your charts, you may end up in a situation where chart upgrades or deletions fail with an error message similar to the following:
+```console
+Error: UPGRADE FAILED: resource mapping not found for name: "<object-name>" namespace: "<object-namespace>" from "": no matches for kind "PodSecurityPolicy" in version "policy/v1beta1"
+ensure CRDs are installed first
+```
+
+This happens as a result of Helm trying to query the cluster for objects that were stored in a previous release data blob.
+To avoid this error, it is required that you clean up these releases that refer to APIs that ceased to exist.
+The Helm project hosts a plugin called `helm-mapkubeapis` that allows you to correct these releases so that a new version can be installed, or a deletion can take place.
+To learn more about the `helm-mapkubeapis` plugin, how it works, and how it can be fine-tuned for your use case, refer to the
+[documentation in GitHub](https://github.com/helm/helm-mapkubeapis#readme).
```

Comment:
```suggestion
This happens when Helm tries to query the cluster for objects that were stored in a previous release's data blob. To clean up these releases and avoid this error, use the `helm-mapkubeapis` Helm  plugin. To learn more about `helm-mapkubeapis`, how it works, and how it can be fine-tuned for your use case, see the [official Helm documentation](https://github.com/helm/helm-mapkubeapis#readme).
```